scripts/tsbs-results-load.py

```diff
@@ -0,0 +1,70 @@
+import csv
+import re
+import json
+import os
+import time
+import psycopg2
+
+resultlist = list()
+with open('tsbsloadoutput.txt', newline='') as csvf:
+    cread = csv.reader(csvf, delimiter=',', quotechar='"')
+    for row in cread:
+        resultlist.append(row)
+del resultlist[0];
+del resultlist[0];
+resultlist.pop();
+resultlist.pop();
+resultlist.pop();
+resultlist.pop();
+
+entry = {};
+entry['type'] = 'tsbsload';
+#entry['repo'] = os.environ.get('GITHUB_REPOSITORY','missing_GITHUB_REPOSITORY');
+entry['repo'] = os.environ.get('DRM_GIT_REPO','missing_DRM_GIT_REPO');
+#entry['ref'] = os.environ.get('GITHUB_REF', 'missing_GITHUB_REF');
+entry['ref'] = os.environ.get('DRM_GIT_BRANCH', 'missing_DRM_GIT_BRANCH');
+#entry['commitid'] = os.environ.get('GITHUB_SHA', 'missing_GITHUB_SHA');
+entry['commitid'] = os.environ.get('DRM_GIT_COMMIT', 'missing_DRM_GIT_COMMIT');
+entry['cpuinfo'] = os.environ.get('CPU_INFO', 'missing_CPU_INFO');
+entry['date'] = time.strftime("%a, %d %b %Y %H:%M:%S +0000", time.gmtime());
+entry['entries'] = {};
+
+start = int(resultlist[0][0]);
+for row in resultlist:
+    print(row);
+    offset = str(int(row[0]) - start);
+    print(offset, ': ', row[4]);
+    entry['entries'][offset] = {};
+    entry['entries'][offset]['rowspersec'] = row[4];
+
+print(json.dumps(entry, indent=2, separators=(',', ': ')));
+
+try:
+    DB_URL = os.environ.get('DB_URL', 'missing_DB_URL');
+    print('db_url length: ', len(DB_URL));
+    conn = psycopg2.connect(DB_URL);
+    cur = conn.cursor();
+
+    cur.execute('select count(*) from githubruns');
+    conn.commit();
+    print(cur.fetchall());
```

Comment:
Probably easier to work with:

```suggestion
    with conn.cursor() as cur;
       cur.execute('select count(*) from githubruns');
       print(cur.fetchall());
```

Also, you should probably read the cursor rows before committing, not after, and there is no reason to commit after each statement. You can just as well do that in a single transaction.